experimental/parser/testdata/parser/method/incomplete.proto.stderr.txt

```diff
@@ -8,13 +8,13 @@ error: missing method return type in service method
   --> testdata/parser/method/incomplete.proto:21:13
    |
 21 |     rpc Bar2(foo.Bar);
-   |             ^^^^^^^^^ expected type in `(...)` after this
+   |             ^^^^^^^^^ expected return type in `returns (...)` after this
 
 error: missing method parameter list in service method
   --> testdata/parser/method/incomplete.proto:22:9
    |
 22 |     rpc Bar3 returns (foo.Bar);
-   |         ^^^^ expected type in `(...)` after this
+   |         ^^^^ expected argument type in `(...)` after this
```

Comment:
I see the "argument type in" vs. "return type in" is intended differentiate the semantic meaning of the thing we're expecting. But it feels awkwardly worded. Above, I suggest just dropping it. But there's another way to rearrange this to make the meaning more clear?